agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -486,9 +489,21 @@ private static void overlayAadEnvVars(Configuration config) {
   }
 
   private static void loadLogCaptureEnvVar(Configuration config) {
-    String loggingEnvVar = getEnvVar(APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL);
-    if (loggingEnvVar != null) {
-      config.instrumentation.logging.level = loggingEnvVar;
+    String loggingLevel = getEnvVar(APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL);
+    if (loggingLevel != null) {
+      configurationLogger.debug(
+          "applying environment variable: {}={}",
+          APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL,
+          loggingLevel);
+      config.instrumentation.logging.level = loggingLevel;
+    }
+    String loggingEnabled = getEnvVar(APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_ENABLED);
```

Comment:
If instrumentation logging is not enabled, setting the instrumentation log level from an environment variable may not be necessary?